tests/random_test.py

```diff
@@ -687,6 +687,12 @@ def testPoissonBatched(self):
     samples = random.poisson(key, lam, shape=(20000,))
     self._CheckChiSquared(samples[:10000], scipy.stats.poisson(2.0).pmf)
     self._CheckChiSquared(samples[10000:], scipy.stats.poisson(20.0).pmf)
+  
+  def testPoissonWithoutShape(self):
+    key = self.seed_prng(1)
+    lam = 2*jnp.ones(10000)
+    samples = random.poisson(key,lam)
+    self._CheckChiSquared(samples,scipy.stats.poisson(2.0).pmf)
```

Comment:
minor: please add a space after commas, and add a space before and after operators like `*`